### PR TITLE
Support EDITNNCR keyword

### PR DESCRIPTION
**Pull request by blattms**
For the provided test case the tansmissibilities on the fault look good. Still the results are a bit different.

Needs OPM/opm-common#3459

### CODE REVIEW

**Comment by blattms** [comment]
We should probably hold off merging this before it is further tested.

build triggered in opm-common

**Comment by blattms** [comment]
:flushed: Embarrassing typo removed.

File: ebos/ecltransmissibility.cc [hunk 1]
```diff
@@ -902,20 +932,20 @@ applyEditNncToGridTrans_(const std::unordered_map<std::size_t,int>& globalToLoca
...
-            const auto& location = nnc_input.edit_location( nnc );
-            auto warning =  fmt::format("Problem with EDITNNC keyword\n"
+    auto print_warning = [&format_ijk, &nncs, &getLocation, &keyword] (const NNCdata& nnc) {
+            const auto& location = getLocation( nnc );
+            auto warning =  fmt::format("Problem with {} keyword\n"
                                         "In {} line {} \n"
-                                        "No NNC defined for connection {} -> {}", location.filename,
+                                        "No NNC defined for connection {} -> {}", keyword, location.filename,
                                         location.lineno, format_ijk(nnc.cell1), format_ijk(nnc.cell2));
             OpmLog::warning("EDITNNC", warning);
```
Comment (atgeirr) [inline ebos/ecltransmissibility.cc:941]: Use `keyword` for tag instead of "EDITNNC".

**Review by atgeirr — COMMENTED** [review]
I think this is fine, with just a minor issue (forgot to change EDITNNC). Will look at the opm-common PR now.

**Comment by blattms** [comment]
jenkins build this with opm-common=3459 please

**Comment by bska** [comment]
jenkins build this please

**Comment by akva2** [comment]
this is holding back rc1.

**Comment by akva2** [comment]
okay. this has already thrown us a day off schedule. i'm calling it. this will not go into rc1.

**Comment by blattms** [comment]
All comments have been addressed a while ago. I will selfmerge this tomorrow.

**Comment by blattms** [comment]
You have been warned...

**Pull request merged**